src/test/java/org/sagebionetworks/bridge/spring/controllers/CacheAdminControllerTest.java

```diff
@@ -60,6 +75,21 @@ public void verifyAnnotations() throws Exception {
     
     @Test
     public void listItems() throws Exception {
+        session.setStudyIdentifier(TEST_STUDY);
+        when(mockAccountDao.getAccount(ACCOUNT_ID)).thenReturn(Account.create());
+        
+        Set<String> items = ImmutableSet.of("A", "B", "C");
+        when(mockCacheAdminService.listItems()).thenReturn(items);
+        
+        // This should be a ResourceList, but it's not currently, so we're maintaining that.
+        Set<String> cacheItems = controller.listItems();
+        assertEquals(items, cacheItems);
+        
+        verify(mockCacheAdminService).listItems();
+    }
+    
+    @Test(expectedExceptions = UnauthorizedException.class)
+    public void listItemsRejectsStudyAdmin() throws Exception {
```

Comment:
Minor: This test has verifies after the call to controller.listItems() tha twill never be called. (Same with removeItemRejectsStudyAdmin().)